### PR TITLE
app-misc/piper: add 0.7 and fix tests

### PR DESCRIPTION
**Pull request by BratishkaErik**

### CODE REVIEW

**Comment by BratishkaErik** [comment]
It is very strange, but I can't reproduce tests failures when building and testing in my home directory (but it is still reproducible with Portage)

**Comment by BratishkaErik** [comment]
> It is very strange, but I can't reproduce tests failures when building and testing in my home directory (but it is still reproducible with Portage)

If I'm running this from root (similar to Portage's invoked command, but added `--verbose` and other directory):
```sh
MALLOC_PERTURB_=130 /usr/bin/flake8 --ignore=E501,W504 /home/bratishkaerik/piper/piper /home/bratishkaerik/piper/piper.in --verbose
```
then it falls with this error:
```python
flake8.checker            MainProcess     81 INFO     Making checkers
flake8.checker            MainProcess     88 INFO     Checking 20 files
flake8.main.application   MainProcess    280 INFO     Finished running
flake8.main.application   MainProcess    280 INFO     Reporting errors
flake8.main.application   MainProcess    281 INFO     Found a total of 109 violations and reported 1
/home/bratishkaerik/piper/piper.in:11:20: E999 SyntaxError: invalid syntax
```
...but if I'm running from my user:
```python
flake8.checker            MainProcess     81 INFO     Making checkers
flake8.checker            MainProcess     88 INFO     Checking 19 files
flake8.main.application   MainProcess    274 INFO     Finished running
flake8.main.application   MainProcess    274 INFO     Reporting errors
flake8.main.application   MainProcess    275 INFO     Found a total of 108 violations and reported 0
```

MYSTICAL...

**Comment by BratishkaErik** [comment]
UPD: Got it, somehow Portage ignores `.flake8` file

File: app-misc/piper/piper-0.7.ebuild [hunk 1]
```diff
@@ -3,27 +3,38 @@
 
 EAPI=8
 
-PYTHON_COMPAT=( python3_{8..10} )
+PYTHON_COMPAT=( python3_{8..11} )
 
 inherit meson python-single-r1 xdg
 
 DESCRIPTION="GTK configuration application for libratbag"
```
Comment (Flowdalic) [inline app-misc/piper/piper-0.7.ebuild:10]: The "official" description on https://github.com/libratbag/piper reads

> GTK application to configure gaming devices

which is more informative, as most people will have no idea what libratbag is.

```suggestion
DESCRIPTION="GTK application to configure gaming devices"
```

Comment (BratishkaErik) [reply app-misc/piper/piper-0.7.ebuild:10]: Thank you! I agree with you, this is way more informative

**Comment by BratishkaErik** [comment]
ping

**Pull request closed without merging**